examples/stm32/xively_demo_mt/Src/system_stm32f4xx.c

```diff
@@ -1,21 +1,21 @@
 /**
   ******************************************************************************
-  * @file    system_stm32f4xx.c

```

Comment:
Yes, this file update is result of updating the SDK - I had compilation error when I used already checked in version of this file. I wonder if we should prepare example per each SDK version ? Or maybe we could use other macro tricks ?